appcheck/firebase-appcheck/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+* [changed] Internal support for `SafetyNet` has been dropped, as `SafetyNet` is no longer being 
```

Comment:
Maybe ".... as the `SafetyNet Attestation API` has been deprecated."

and also link to https://developer.android.com/privacy-and-security/safetynet/deprecation-timeline#safetynet_attestation_deprecation_timeline here?